src/wp-includes/class-wp-term.php

```diff
@@ -182,8 +184,9 @@ public static function get_instance( $term_id, $taxonomy = null ) {
 		}
 
 		$term_obj = new WP_Term( $_term );
-		$term_obj->filter( $term_obj->filter );
-
+		if ( $filter ) {
```

Comment:
Do we need to check if `$filter` should be `row` then it will allow for sanitize?

```suggestion
		if ( 'raw' === $filter ) {
```